src/permissions.ts

```diff
@@ -92,61 +92,4 @@ export async function salesStaff(from: RouteLocationNormalized,to: RouteLocation
     } else {
       next('/404')
     }
-  }
-
-  export async function isManager(){
-    const allowedRoles = ['SUPER_ADMIN', 'ADMIN'];
-    const allowedCompanies = ['PARENT', 'STORE', 'RETIAL']
-    await store.dispatch(ActionTypes.USER_DATA);
-    const role = store.getters.getUser ? store.getters.getUser.user_type : '';
-    const company_type = store.getters.getUser && store.getters.getUser.company ? store.getters.getUser.company.company_type : '';
-    if (role != null &&
-      company_type != null &&
-      company_type != '' &&
-      role != '' &&
-      allowedCompanies.includes(company_type) &&
-      allowedRoles.includes(role)){
-        debugger
```

Comment:
Why removed this code?

Reply:
These boolean functions were created in permissions.ts for reusability. But didn't work when called inside components.